dev-test/codegen.ts

```diff
@@ -98,12 +98,6 @@ const config: CodegenConfig = {
       config: { enumsAsTypes: true },
       plugins: ['typescript', 'typescript-operations'],
     },
-    './dev-test/githunt/types.d.ts': {
```

Comment:
if i'm understanding this file correctly, this is a dupe of the other file (in other words, 
 `./dev-test/githunt/types.enumsAsTypes.ts` and  `./dev-test/githunt/types.d.ts` are the same) - here is the config that we keeping :
```
    './dev-test/githunt/types.enumsAsTypes.ts': {
      schema: './dev-test/githunt/schema.json',
      documents: './dev-test/githunt/**/*.graphql',
      config: { enumsAsTypes: true },
      plugins: ['typescript', 'typescript-operations'],
    },
```

so, I got rid of this one because `enumsAsTypes` no longer outputs `.d.ts` files. if you have consts in .d.ts files, you'd get an error that looks like:
```
TS1254: A 'const' initializer in an ambient context must be a string or numeric literal or literal enum reference
```

@saihaj, will defer to you on the severity of that change.

Reply:
if we add as a new options https://github.com/dotansimha/graphql-code-generator/pull/8746#pullrequestreview-1248379596 then I guess we can keep this and also add new tests 🤔